Test Project/MVVM/View/ViewController.swift

```diff
@@ -16,7 +16,8 @@ class ViewController: UIViewController {
     @IBOutlet weak var collectionViewContentList: UICollectionView!
     
     var storedOffsets = [Int: CGFloat]()
-    var contentList: ContentListModel?
+    var contentListViewModel = ContentListViewModel()
+    var contentListData: ContentListModel?
```

Comment:
Try moving data into viewModel. Controllers should only contain the views. not data models.